pkg/apis/scaler/v1alpha1/crd_demand.go

```diff
@@ -49,109 +49,118 @@ var (
 	demandGroupResource        = demandGroupVersionResource.GroupResource()
 	oneFloat                   = float64(1)
 	oneInt                     = int64(1)
-	demandDefinition           = v1beta1.CustomResourceDefinition{
+	versionDefinition          = v1.CustomResourceDefinitionVersion{
+		Name:    SchemeGroupVersion.Version,
+		Served:  true,
+		Storage: true,
+		Schema: &v1.CustomResourceValidation{
+			OpenAPIV3Schema: &v1.JSONSchemaProps{
+				Type:     "object",
+				Required: []string{"spec", "metadata"},
+				Properties: map[string]v1.JSONSchemaProps{
+					"status": {
+						Type:     "object",
+						Required: []string{"phase"},
+						Properties: map[string]v1.JSONSchemaProps{
+							"phase": {
+								Type: "string",
+								Enum: getAllowedDemandPhasesEnum(),
+							},
+							"last-transition-time": {
+								Type:     "string",
+								Format:   "date-time",
+								Nullable: true,
+							},
+						},
+					},
+					"spec": {
+						Type:     "object",
+						Required: []string{"units", "instance-group"},
+						Properties: map[string]v1.JSONSchemaProps{
+							"instance-group": {
+								Type:      "string",
+								MinLength: &oneInt,
+							},
+							"is-long-lived": {
+								Type: "boolean",
+							},
+							"units": {
+								Type: "array",
+								Items: &v1.JSONSchemaPropsOrArray{
+									Schema: &v1.JSONSchemaProps{
+										Type:     "object",
+										Required: []string{"count", "cpu", "memory"},
+										Properties: map[string]v1.JSONSchemaProps{
+											"count":  {Type: "integer", Minimum: &oneFloat},
+											"cpu":    {Type: "string", MinLength: &oneInt},
+											"memory": {Type: "string", MinLength: &oneInt},
+											"gpu":    {Type: "string", MinLength: &oneInt},
```

Comment:
We won't support GPUs in v1alpha1 no?

Reply:
see description, we need to add it for the conversion code.